app/lib/common/pages/fatal_fail.dart

```diff
@@ -0,0 +1,58 @@
+
+import 'package:acter/common/dialogs/nuke_confirmation.dart';
+import 'package:acter/common/snackbars/custom_msg.dart';
+import 'package:acter/common/utils/routes.dart';
+import 'package:atlas_icons/atlas_icons.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:go_router/go_router.dart';
+
+class FatalFailPage extends ConsumerWidget {
```

Comment:
Is this page supposed to work as retry for nuke action ? Or a common routing failure fallback?

Reply:
This shows up on a really bad failure, usually when we try to start up and the rust SDK fails to initialize. Hence it featuring the nuke-option... _usually_ this isn't something a regular end-user sees, just us devs when we moved the app/database into an unrecoverable state...

Reply:
No, I can make a sense of it now, just that nuke option on the page seemed odd to me.